src/components/investigate-dialog.tsx

```diff
@@ -0,0 +1,62 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./investigate-dialog.sass";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class InvestigateDialogComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="investigate-dialog">
+        <div className="investigate-dialog-header">
+          <div className="investigate-dialog-title">Investigate</div>
+          <div className="investigate-dialog-close" onClick={this.handleClickClose}>x</div>
+        </div>
+        <div className="investigate-dialog-content">
+          <div className="investigate-dialog-message">Select a [space] to begin your investigation</div>
+          <div className="investigate-dialog-flex-container">
+            <div className="investigate-dialog-button-holder" onClick={this.handleClickInvestigatePopulation}>
+              <div className="investigate-dialog-button">Population</div>
+              <div className="investigate-dialog-button-footer">Some descriptive text goes into this space</div>
+            </div>
+            <div className="investigate-dialog-button-holder">
+              <div className="investigate-dialog-button">Breeding</div>
+              <div className="investigate-dialog-button-footer">Some descriptive text goes into this space</div>
+            </div>
+            <div className="investigate-dialog-button-holder">
+              <div className="investigate-dialog-button">Scope/Zoom</div>
+              <div className="investigate-dialog-button-footer">Some descriptive text goes into this space</div>
+            </div>
+            <div className="investigate-dialog-button-holder">
+              <div className="investigate-dialog-button">Protein/DNA</div>
+              <div className="investigate-dialog-button-footer">Some descriptive text goes into this space</div>
+            </div>
+            <div className="investigate-dialog-button-holder">
+              <div className="investigate-dialog-button">Comparison</div>
+              <div className="investigate-dialog-button-footer">Some descriptive text goes into this space</div>
+            </div>
+          </div>
+        </div>
+        <div className="investigate-dialog-footer"/>
+      </div>
+    );
+  }
+
+  private handleClickClose = () => {
+    const {ui} = this.stores;
+    { ui.setShowInvestigationModalSelect(!ui.showInvestigationModalSelect); }
```

Comment:
Braces here and below seem unnecessary.

Reply:
I must not be understanding proper syntax.  It throws and error without the braces.

Reply:
Interesting, it compiles on my end - I just pushed up a commit and it seems like Travis handled it fine too. 

Braces are used in React to run code in a JSX object. So I could say something like this:

```
render() {
  return (
    const className = "name"
    <div className={className}>
      {"Welcome " + props.user ? urlParams.user : "User"}
      {props.showThing && <div>thing</div>}
    </div>
  )
}

Reply:
Are we looking at different lines?  I'll look at your commit.

Reply:
Bleh.  I thought you meant here:
`const {ui} = this.stores;`

Reply:
Yup.  Works fine.

Reply:
And those braces, I assume, are needed as part of the MST syntax?

Reply:
Basic TS/JS destructuring -- not MST.

Reply:
Ah, yes! Sorry I didn't specify. And right, as @kswenson said:

`const {ui} = this.stores;`

is equivalent to:

`const ui = this.stores.ui;`

Reply:
Ah yes, sorry my comment wasn't more specific!

Right, what Kirk said - basically this:

`const {ui} = this.stores;`

is equivalent to this:

`const ui = this.stores.ui;`